### PR TITLE
Bump `elm` to `0.19.1`

### PR DESCRIPTION
**Pull request by jeffwidman**
Bump to https://github.com/elm/compiler/releases/tag/0.19.1 to pickup some bugfixes.

### CODE REVIEW

**Review by landongrindheim — APPROVED** [review]
I think this makes sense, but it's also worth pointing out that we may not have many resources to support any regressions that might come with this version bump. Looks good to me!

File: elm/Dockerfile [hunk 1]
```diff
@@ -6,10 +6,9 @@ ARG TARGETARCH
 # - https://github.com/elm/compiler/issues/2232
 ENV PATH="$PATH:/node_modules/.bin"
 RUN [ "$TARGETARCH" != "amd64" ] \
-  || (curl -sSLfO "https://github.com/elm/compiler/releases/download/0.19.0/binaries-for-linux.tar.gz" \
-  && tar xzf binaries-for-linux.tar.gz \
-  && mv elm /usr/local/bin/elm19 \
-  && rm -f binaries-for-linux.tar.gz)
```
Comment (jeffwidman) [inline elm/Dockerfile:12]: No need to remove anymore as `gunzip` deletes the zipped file.

**Comment by jeffwidman** [comment]
I threw this up quickly in passing, but can't afford to spend a lot of time on it right now.

I was expecting this to be a straightforward bump, but looks like there's some additional validation and the error messages changed.

Given this is our least popular ecosystem, the user impact isn't high enough for me to justify taking this one across the finish line... but if someone from the community wants to step up and finish this, I'd be happy to review/merge it.

**Comment by jeffwidman** [comment]
I'm going to close this, but again, if anyone from the community wants to take a stab at pushing it across the finish line, feel free! 🎉

**Pull request closed without merging**